### ISSUE TITLE
use auth action with Ansible 

### ISSUE THREAD

**Issue by kaleocheng** [teleport-actions/root#23]
I'm trying to use  machine id + github action by following  https://goteleport.com/docs/ver/11.x/machine-id/guides/github-actions/  which works with those:

```yaml
    - name: Fetch credentials using Machine ID
      id: auth
      uses: teleport-actions/auth@v1
      with:
        # Use the address of the auth/proxy server for your own cluster.
        proxy: example.domain:443
        # Use the name of the join token resource you created in step 1.
        token: github-token
        # Specify the length of time that the generated credentials should be
        # valid for. This is optional and defaults to "1h"
        certificate-ttl: 1h

    - name: list nodes
      run: tsh -i ${{ steps.auth.outputs.identity-file }} ls 
```

but when I try to use Ansible based on that:

```yaml
    - name: Fetch credentials using Machine ID
      id: auth
      uses: teleport-actions/auth@v1
      with:
        # Use the address of the auth/proxy server for your own cluster.
        proxy: example.domain:443
        # Use the name of the join token resource you created in step 1.
        token: github-token
        # Specify the length of time that the generated credentials should be
        # valid for. This is optional and defaults to "1h"
        certificate-ttl: 1h

    - name: Generate ssh.cfg
      run: tsh -i ${{ steps.auth.outputs.identity-file }} config > ssh.cfg

    - name: Run Ansible Playbook
      run: |
        ansible-playbook -vv -i inventory -u devops \
          --ssh-extra-args "-F ssh.cfg" \
          ping.yaml
```

I got key not found:

```
    debug1: Next authentication method: publickey
    debug1: Trying private key: /home/admin/.tsh/keys/example.domain/bot-1
    no such identity: /home/admin/.tsh/keys/example.domain/bot-1: No such file or directory
    debug2: we did not send a packet, disable method
    debug1: No more authentication methods to try.
    devops@test.teleport-admin: Permission denied (publickey).
  unreachable: true

PLAY RECAP *********************************************************************
test.teleport-admin     : ok=0    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0  
```

the path `/home/admin/.tsh/keys/example.domain/bot-1` is indeed from `tsh config` above.  I assume it didn't put the real key in there when I run the auth with Github Action.

any suggest for that?

**Comment by strideynet** [comment]
It looks like using an identity file with `tsh config` does not generate a correct config file (the resulting config file does not include the `-i`).

I'll make a change to expose the SSH config generated by `tbot`.

**Comment by strideynet** [comment]
I've released v1.0.3 which includes a new output.

You should be able to do ```
--ssh-extra-args "-F ${{ steps.auth.outputs.ssh-config }}"
```
